helpers/osinfo_test.go

```diff
@@ -0,0 +1,164 @@
+package helpers
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOSReleaseID_String(t *testing.T) {
```

Comment:
only tests for the existence of value in map, i don't feel this is necessary. will remove if not needed, else will add `TestOSReleaseField_String` later.